lib/openapi_client.rb

```diff
@@ -11,59 +11,66 @@
-require_relative 'openapi_client/models/web_authn_icons'
-require_relative 'openapi_client/models/web_authn_type'
+require 'openapi_client/models/app_info'
+require 'openapi_client/models/app_response'
+require 'openapi_client/models/apple_user_social_connection'
+require 'openapi_client/models/auth_methods'
+require 'openapi_client/models/create_magic_link_request'
+require 'openapi_client/models/create_user_request'
+require 'openapi_client/models/element_customization'
+require 'openapi_client/models/font_family'
+require 'openapi_client/models/github_user_social_connection'
+require 'openapi_client/models/google_user_social_connection'
+require 'openapi_client/models/layout_config'
+require 'openapi_client/models/layouts'
+require 'openapi_client/models/link'
+require 'openapi_client/models/list_devices_response'
+require 'openapi_client/models/list_paginated_users_item'
+require 'openapi_client/models/list_paginated_users_response'
+require 'openapi_client/models/magic_link'
+require 'openapi_client/models/magic_link_auth_method'
+require 'openapi_client/models/magic_link_channel'
+require 'openapi_client/models/magic_link_response'
+require 'openapi_client/models/magic_link_type'
+require 'openapi_client/models/model400_error'
+require 'openapi_client/models/model401_error'
+require 'openapi_client/models/model403_error'
+require 'openapi_client/models/model404_error'
+require 'openapi_client/models/model500_error'
+require 'openapi_client/models/nonce'
+require 'openapi_client/models/otp_auth_method'
+require 'openapi_client/models/paginated_links'
+require 'openapi_client/models/passkeys_auth_method'
+require 'openapi_client/models/technologies'
+require 'openapi_client/models/ttl_display_unit'
+require 'openapi_client/models/update_magic_link_auth_method'
+require 'openapi_client/models/update_otp_auth_method'
+require 'openapi_client/models/update_passkeys_auth_method'
+require 'openapi_client/models/update_user_request'
+require 'openapi_client/models/user_event_status'
+require 'openapi_client/models/user_info'
+require 'openapi_client/models/user_metadata_field'
+require 'openapi_client/models/user_metadata_field_type'
+require 'openapi_client/models/user_recent_event'
+require 'openapi_client/models/user_response'
+require 'openapi_client/models/user_social_connections'
+require 'openapi_client/models/user_status'
+require 'openapi_client/models/web_authn_devices'
+require 'openapi_client/models/web_authn_icons'
+require 'openapi_client/models/web_authn_type'
 
 # APIs
-require_relative 'openapi_client/api/apps_api'
-require_relative 'openapi_client/api/magic_links_api'
-require_relative 'openapi_client/api/tokens_api'
-require_relative 'openapi_client/api/user_devices_api'
-require_relative 'openapi_client/api/users_api'
+require 'openapi_client/api/apps_api'
+require 'openapi_client/api/magic_links_api'
+require 'openapi_client/api/tokens_api'
+require 'openapi_client/api/user_devices_api'
+require 'openapi_client/api/users_api'
```

Comment:
Failing because of the error in this file. they all need to be `require_relative`

Reply:
Info here: https://github.com/passageidentity/passage-ruby/issues/57